package.json

```diff
@@ -26,6 +26,7 @@
 	    "SmartStore.podspec",
         "SmartSync.podspec",
         "SalesforceReact.podspec",
+        "SalesforceSwiftSDK.podspec",
```

Comment:
We don't pull SalesforceMobileSDK-iOS into our generated app through npm anymore but I thought we should still update the package.json.